views/bonfire/show.jade

```diff
@@ -143,5 +149,9 @@ block content
                     $('.btn-twitter').attr('href', url);
                 }
         );
+        var MDNlinks = !{JSON.stringify(MDNlinks)};
```

Comment:
It tells jade to to turn MDNlinks into a string, then writes the output to this section of the file as text.